src/components/elements/CommonDialog.tsx

```diff
@@ -0,0 +1,26 @@
+import CloseIcon from '@mui/icons-material/Close';
+import { Dialog, DialogProps, IconButton } from '@mui/material';
+
+const CommonDialog = ({ children, ...props }: DialogProps) => {
+  return (
+    <Dialog {...props}>
+      {children}
+      {props.onClose && (
+        <IconButton
+          aria-label='close'
+          onClick={(e) => props.onClose && props.onClose(e, 'backdropClick')}
+          sx={{
+            position: 'absolute',
```

Comment:
It seems possible for the dialog title to flow under the close button. Perhaps there should be some left padding on the title?

Reply:
Added!